recipes/xxhash/all/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.4)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
```

Comment:
```suggestion
conan_basic_setup()
```

Reply:
I'd say targets are better, as they use so-called "modern CMake" (don't like the name, but anyway)

Reply:
there are no dependencies, so no target to create